docs/interpreter/cassandra.md

```diff
@@ -789,5 +811,7 @@ Below are the configuration parameters and their default value.
 [standard CQL syntax]: http://docs.datastax.com/en/cql/3.1/cql/cql_using/use_collections_c.html
 [Tuple CQL syntax]: http://docs.datastax.com/en/cql/3.1/cql/cql_reference/tupleType.html
 [UDT CQL syntax]: http://docs.datastax.com/en/cql/3.1/cql/cql_using/cqlUseUDT.html
+[Zeppelin dynamic form]: http://zeppelin.incubator.apache.org/docs/latest/manual/dynamicform.html
+[Interpreter Binding Mode]: http://zeppelin.incubator.apache.org/docs/0.6.0-incubating-SNAPSHOT/manual/interpreters.html
```

Comment:
@doanduyhai Ah right. Is there any specific reason not to use `latest` instead of  `0.6.0-incubating-SNAPSHOT` ?